### ISSUE TITLE
3 errors in test_make_model.py

### ISSUE THREAD

**Issue by epogrebnyak** [epogrebnyak/make-xls-model#34]
`py.test test_make_model.py` on branch `test_and_simplify_make` now has 3 errors, must clean tests.

**Comment by arwer13** [comment]
The errors may be fixed by removing `is_forecast` rows from `data` and `controls` sheets in `spec.xls`.
There are lines in `make_xl_model.py`

```
data_df[IS_FORECAST_LABEL] = 0 
controls_df[IS_FORECAST_LABEL] = 1
```

so the application doesn't actually expect the `is_forecast` rows to be there, and the values are overwritten.
I suggest just to remove the rows and not to modify logic at the moment.  The rows are redundant, and for the new fused data-control sheet there would be another solution.

**Comment by epogrebnyak** [comment]
I agree, thanks for documenting the problem in detail.
25 февр. 2016 г. 9:28 пользователь "Veremeenko Artyom" <
notifications@github.com> написал:

**Comment by epogrebnyak** [comment]
Fixed and closed by e9cbde3e8a590a282dd063bc5f10c5a569ca0180
